pipelines/google_sheets/data_validator.py

```diff
@@ -3,14 +3,16 @@
 from dlt.common.typing import DictStrAny
 from datetime import datetime, timedelta
 from re import match
+from dlt.common import pendulum
 
-
-# Date constant- n.o days from 1st Jan 1AD to 30th December 1899
-SERIAL_NUMBER_ORIGIN_ORDINAL = 693594
 # this string comes before the id
 URL_ID_IDENTIFIER = "d"
 # time info
 SECONDS_IN_DAY = 86400
+# TIMEZONE info
+DLT_TIMEZONE = "UTC"
```

Comment:
pendulum provided by `dlt` should have UTC by default. does it make any difference when you remove that?